### DISCUSSION TITLE
[Feature Request] Display Context Menu without the labels

### DISCUSSION THREAD

**Discussion by himanshu1721** [nandorojo/zeego#121] [Ideas]
Description: 
- It would be helpful for the developer to be able to add the context menu without the labels i.e. the labels in the Context menu being optional.

Problem statement: 
- There might be cases for the developer to implement the context menu without displaying the labels as shown in the below screenshot. The purpose for it could be for aesthetic reasons or emphasize a particular component in the UI.

![Screenshot 2024-12-10 at 11 47 08 PM 1 (1)](https://github.com/user-attachments/assets/3f1560bb-5130-4237-8cf9-f19c8d758ddf)

![Group 142](https://github.com/user-attachments/assets/2fcd97b9-ec67-4c1a-9b3a-b143a2883ad9)

**Comment by himanshu1721** [comment]
@nandorojo Hey Fernando. please have a look.

**Comment by nandorojo** [comment]
Hey no need to tag on this. You can open an issue on the iOS library if you'd like. It's not something Zeego can do if that library doesn't support it
